src/config/inversion-fixes.config

```diff
@@ -183,6 +183,13 @@ div.orb-nav-section.orb-nav-blocks > a > img
 
 ================================
 
+bestbuy.com
+
+INVERT
+#None > div > section > div.comparison-container.comparison-container-l.legacy-variant
```

Comment:
Let's add `.product-container` So the product itself also becomes inverted.
![](https://i.imgur.com/2uW34zN.png)

Reply:
So in effect, inverting once for the `.comparison-container`, then inverting again on just the `.product-container`.

Thank you for the help.

I will update the branch and PR.